python/cudf/cudf/core/series.py

```diff
@@ -1730,27 +1730,47 @@ def count(self, axis=None, skipna=True):
         assert axis in (None, 0) and skipna is True
         return self.valid_count
 
-    def min(self, axis=None, skipna=True, dtype=None):
+    def min(self, axis=None, skipna=True, dtype=None, **kwargs):
         """Compute the min of the series
         """
         assert axis in (None, 0) and skipna is True
-        return self.nans_to_nulls().dropna()._column.min(dtype=dtype)
+        nan_as_null = kwargs.get("nan_as_null", True)
```

Comment:
pandas bakes both `NA` and `NaN` into the `skipna` flag as of 1.0 so we should just do the same.

Reply:
@kkraus14 Removed `nan_as_null` flag and using `skipna` to drop nulls.